pkg/osquery/tables/dataflattentable/tables.go

```diff
@@ -39,6 +39,16 @@ type Table struct {
 	keyValueSeparator string
 }
 
+// AllTablePlugins is a helper to return all the expected flattening tables.
+func AllTablePlugins(client *osquery.ExtensionManagerClient, logger log.Logger) []*table.Plugin {
```

Comment:
these are the generic dataflatten tables that extract data from a file on disk, correct?

Reply:
Yep.